src/Components/Web.JS/package.json

```diff
@@ -4,6 +4,7 @@
   "version": "0.0.1",
   "description": "",
   "main": "index.js",
+  "sideEffects": false,
```

Comment:
If I understand [the meaning of this property](https://webpack.js.org/guides/tree-shaking/#clarifying-tree-shaking-and-sideeffects) correctly, it's not really accurate to say `sideEffects: false` since this isn't a pure module. It does have side effects, such as triggering the loading of Blazor apps, and the placement of APIs in global scope (e.g., `Blazor.*` and Emscripten's `Module.*`).

If this flag causes a meaningful reduction in size while not breaking functionality, then I'd be fine with lying to Webpack here, since we should be able to notice if anything new becomes broken in the future.

Reply:
Oh, I'm a bit confused now because https://github.com/dotnet/aspnetcore/pull/30320/files#diff-d40149c3a278492570fd82822aa94771c9fe60671c33a9508fa3576f0d9aee02R24 says `sideEffects: true`. Is that the real source of truth? If so, can this line be removed from `package.json`, or does it serve a different purpose?

Reply:
`sideEffects` is more action at a distance; if one method can effect another method without any obvious coupling, which is one of the reasons I moved to an imported `Blazor` object rather than disparate method assignments on `window["Blazor"]` which are then called in uncoupled methods. i.e. "must be included" code, regardless of whether its called.

Might be getting away with it as the side effect methods are in the main startup path; however just in case added 
```ts
"sideEffects": [
  "src/GlobalExports.ts",
  "src/Platform/Mono/MonoPlatform.ts"
],
```

No change on overall size.

>  says `sideEffects: true` ...

That one is `optimization` object which says to use side effect processing in tree shaking